libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -434,7 +448,8 @@ void AP_Mount::init()
     for (uint8_t instance=0; instance<AP_MOUNT_MAX_INSTANCES; instance++) {
         // default instance's state
         state[instance]._mode = (enum MAV_MOUNT_MODE)state[instance]._default_mode.get();
-
+        state[instance]._rcinvalid_mode = (enum MAV_MOUNT_MODE)state[instance]._default_rcinvalid_mode.get();
```

Comment:
Why do we need this sideways-copy?

Reply:
monkey see monkey do...will try to eliminate it

Reply:
fixed....